### PR TITLE
Update to Ember 3.14

### PR DESCRIPTION
**Pull request by algodave**
Updates gradually to EmberJS 3.14.0

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,12 +49,11 @@ jobs:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
     - stage: "Additional Tests"
-      env: EMBER_TRY_SCENARIO=ember-lts-2.16
```
Comment (Turbo87) [inline .travis.yml]: does that mean Ember 2.16 is no longer supported? in that case this probably should be considered a breaking change and cause a new major release 🤔

File: package.json [hunk 2]
```diff
@@ -29,38 +29,38 @@
     "url": "https://github.com/algonauti/ember-g-recaptcha/issues"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.16.0"
+    "ember-cli-babel": "^7.11.1",
```
Comment (Turbo87) [inline package.json]: this change means that we now need a minimum Ember CLI version of 2.13, which should probably also be considered a breaking change

File: package.json [hunk 3]
```diff
@@ -29,38 +29,38 @@
...
+    "eslint-plugin-ember": "^7.1.0",
+    "eslint-plugin-node": "^10.0.0",
     "loader.js": "^4.7.0",
-    "qunit-dom": "^0.7.1",
+    "qunit-dom": "^0.9.0",
     "sweetalert": "~1.1.3"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || >= 10.*"
```
Comment (Turbo87) [inline package.json]: this is also a breaking change since we're dropping support for Node 6

File: package.json (hunk 2 above)
Comment (algodave) [reply package.json]: The disclaimer "This addon only supports Ember 3 or above" has been in the Readme since version 1.0.0

**Pull request merged**